resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,86 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('has_disabled_link_warning') === 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        :href="hasDisabledLinkWarning() ? '{{ $url }}' : 'javascript:;'"
+        :target="hasDisabledLinkWarning() ? '_blank' : '_self'"
```

Comment:
why `_self`?

Reply:
To open in the same window, it's the default behavior

Reply:
but why? it's an external link so might as well keep it `_blank` at all times?

Reply:
Had to go back on it to remember, but no, issue is that if I leave it as `target="_blank"` it will open a new empty tab whenever we open up the modal, meanwhile, if we specify `target="_self"` it works as intended. Some weird quirk I guess ?